features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -91,6 +91,7 @@ export function SidebarNavigation() {
               isCollapsed={isSidebarCollapsed}
               onClick={() => toggleSidebar()}
               className={styles.collapseMenuItem}
+              flipped={isSidebarCollapsed}
```

Comment:
The problem I see with adding a new prop here is that it is only used to customize one specific element. Here you could rotate the icon by adding styles to the `collapseMenuItem` class. This is already used to apply custom styles. You could either target the image or import the `styles.icon` class from `menu-item-link.module.scss`.

Not a big deal of course, but adding props to customize components for a single use case easily leads to cluttered props. That in turn makes the component difficult to understand and use.